spec/models/scheme_spec.rb

```diff
@@ -209,4 +209,16 @@
       end
     end
   end
+
+  describe "owning organisation" do
+    let(:stock_owning_org) { FactoryBot.create(:organisation, holds_own_stock: true) }
+    let(:non_stock_owning_org) { FactoryBot.create(:organisation, holds_own_stock: false) }
+    let(:scheme) { FactoryBot.create(:scheme, owning_organisation_id: stock_owning_org.id) }
+
+    context "when the owning organisation is set as a non-stock-owning organisation" do
+      it "throws the correct validation error" do
+        expect { scheme.update!({ owning_organisation_id: non_stock_owning_org.id }) }.to raise_error(ActiveRecord::RecordInvalid, /#{I18n.t("validations.scheme.owning_organisation.does_not_own_stock")}/)
```

Comment:
How do you know that this is throwing the correct validation error?

Reply:
When I change the expected validation (e.g. to `activerecord.errors.models.scheme.attributes.owning_organisation_id.invalid`) the test fails:

`expected ActiveRecord::RecordInvalid with message matching /Enter the name of the organisation that owns the housing stock/, got #<ActiveRecord::RecordInvalid: Validation failed: Owning organisation Enter an organisation that owns housing stock>`